app/controllers/sessions_controller.rb

```diff
@@ -8,8 +8,11 @@ def new
   def create
     session[:auth_user] = auth_hash
 
+    Raven.tags_context(sign_in_user_id: current_user.fetch('uid'))
```

Comment:
`skip_before_action :authenticate` causes `current_user` to not get set here, so I don't think this does anything but set these to `nil`. I'm surprised this doesn't blow up actually.

Reply:
If I'm not mistaken, `current_user` just pulls out `session[:auth_user]` (https://github.com/DFE-Digital/manage-courses-frontend/pull/257/files#diff-55c5b7aecfb519d0e4880eaf2788eb6eR23), so once that is set it's save to access `current_user`.